### DISCUSSION TITLE
Api to Trigger all Dags

### DISCUSSION THREAD

**Discussion by msoni1369** [apache/airflow#32024] [General]
Hi Everyone, i have a use case where i have to trigger all dags at airflow endpoint or dags with similar name can solve my problem as well. I checked the trigger api documentation, looks like trigger api does not support regular expression also there is no api which trigger all dags at once. Can anyone help me on this.

**Comment by potiuk** [comment]
Iterate over all dags and trigger those that would match your regexp. That's about 10 lines of Python code for yout to write.
